<ui version="4.0" >
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>129</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" >
   <item row="4" column="0" >
    <widget class="QWidget" native="1" name="installFrame" >
     <layout class="QGridLayout" name="gridLayout_2" >
      <property name="margin" >
       <number>0</number>
      </property>
      <item row="1" column="0" colspan="2" >
       <widget class="QProgressBar" name="installationProgress" >
        <property name="value" >
         <number>24</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2" >
       <widget class="QWidget" native="1" name="konsoleFrame" />
      </item>
      <item row="4" column="0" colspan="2" >
       <spacer name="verticalSpacer" >
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>397</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" colspan="2" >
       <widget class="QLabel" name="installingLabel" >
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="titleLabel" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="0" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>271</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel" >
     <x>271</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>271</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel" >
     <x>271</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
